rocksdb/rocksdb.nim

```diff
@@ -63,6 +63,49 @@ type
     writeOpts: WriteOptionsRef
     ingestOptsPtr: IngestExternalFilesOptionsPtr
 
+proc listColumnFamilies*(
+    path: string;
+    dbOpts = defaultDbOptions();
```

Comment:
the dboptions here also need freeing - thus, it's better not to provide a default

Reply:
Ok, can roll that out with this one

Reply:
same with `defaultReadOptions()` etc.